src/selectors/nodes.js

```diff
@@ -232,3 +235,29 @@ export const getVisibleNodes = createSelector(
         }))
       : []
 );
+
+export const getNodeswithInputParams = createSelector(
```

Comment:
Could you add a JS doc here (similar to others in this file)? Also use camel casing on the name